utils/build-script-impl

```diff
@@ -1491,19 +1516,26 @@ function set_swiftpm_bootstrap_command() {
 
 
 for host in "${ALL_HOSTS[@]}"; do
+    # Skip this pass when the only action to execute can't match.
+    if [[ "${ONLY_EXECUTE}" != "" ]] && ! [[ "${ONLY_EXECUTE}" == ${host}-*-build ]]; then
```

Comment:
I factored out a helper function to make the logic more obvious, and made the ! [[ == ]] change.